docs/components/aws/elasticache.md

```diff
@@ -0,0 +1,792 @@
+---
+description: AWS ElastiCache, in-memory data storage and cache services for your serverless functions, with SLAppForge Sigma cloud IDE
+---
+
+# ![](images/elasticache/ecache_icon.svg) AWS ElastiCache
+
+ElastiCache is a fully managed, in-memory data store for enterprise-level, data-intensive applications. ElastiCache 
+can improve the performance of consumer applications, by retrieving data from its in-memory data stores with high throughput and low latency. 
+ElastiCache supports two open-source cache engines, Amazon ElastiCache for Redis and Amazon ElastiCache for Memcached.
```

Comment:
, Redis and Memcached

(drop "Amazon ElastiCache for " part coz that's not part of the open-source engine name)